sceptre/cli/describe.py

```diff
@@ -31,13 +32,20 @@ def describe_change_set(ctx, path, change_set_name, verbose):
     Describes the change set.
 
     """
-    stack, _ = get_stack_or_stack_group(ctx, path)
+    context = SceptreContext(
+                command_path=path,
+                project_path=ctx.obj.get("project_path", None),
+                user_variables=ctx.obj.get("user_variables", {}),
+                options=ctx.obj.get("options", {})
+            )
+
+    stack, _ = get_stack_or_stack_group(context)
     action = 'describe_change_set'
-    plan = SceptrePlan(path, action, stack)
+    plan = SceptrePlan(context, action, stack)
     description = plan.execute(change_set_name)
     if not verbose:
         description = simplify_change_set_description(description)
-    write(description, ctx.obj["output_format"])
+    write(description, context.obj["output_format"])
```

Comment:
This should be `context.output_format` and I believe you need to pass it in the context above as well.

Reply:
now be addressed